app/controllers/api/account.php

```diff
@@ -1677,6 +1677,96 @@
         ])]), Response::MODEL_JWT);
     });
 
+App::post('/v1/account/targets')
+    ->desc('Create Account Target')
+    ->groups(['api', 'account'])
+    ->label('error', __DIR__ . '/../../views/general/error.phtml')
+    ->label('audits.event', 'target.create')
+    ->label('audits.resource', 'target/response.$id')
+    ->label('event', 'users.[userId].targets.[targetId].create')
+    ->label('scope', 'public')
+    ->label('docs', false)
+    ->param('targetId', '', new CustomId(), 'Target ID. Choose a custom ID or generate a random ID with `ID.unique()`. Valid chars are a-z, A-Z, 0-9, period, hyphen, and underscore. Can\'t start with a special char. Max length is 36 chars.')
+    ->param('providerType', '', new WhiteList(['email', 'sms', 'push']), 'The target provider type. Can be one of the following: `email`, `sms` or `push`.')
```

Comment:
We can remove this param once routes are push-specific

Reply:
removed providerType